### DISCUSSION TITLE
New Feature: submit ToS and PP urls to ToSBack

### DISCUSSION THREAD

**Discussion by hugoroy** [tosdr/tosdr.org#213] [Ideas]
For now I made: https://tosdr.org/submit-tosback.html

**Comment by Ryuno-Ki** [comment]
How is that different from the site in #4?

**Comment by hugoroy** [comment]
See https://tosback.org and tosback repositories.

**Comment by Ryuno-Ki** [comment]
From their site: <blockquote>Every day, we check the Terms and Policies of many online services to see if any of them have changed</blockquote>

I'd do that with a script checking md5sums against :)
How can I highlight @EFForg here? That is, draw the person's attention of them, which works together with TOS-DR at this.

**Comment by hugoroy** [comment]
This isn't as easy as you think. @JimmStout is working on it. Anyway, this is off topic :-)

**Comment by cauerego** [comment]
i never used gatsbyjs or 5apps, but unless you can put them both together i really do think it's worth considering another host. (gatsbyjs will probably be my next main website tool, for many reasons... here i just learned a new one, that) building a form should be trivial:

https://www.gatsbyjs.com/docs/building-a-contact-form/

**Comment by weisk** [comment]
Hi guys, may I be of any service to you?
